CHANGELOG.md

```diff
@@ -79,6 +80,84 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.13.0
+
+### Change
+
+- refactor(ai-proxy): move read_response into ai_driver.request function [#12101](https://github.com/apache/apisix/pull/12101)
+- refactor: mcp server framework implementation #[12168](https://github.com/apache/apisix/pull/12168)
+- change: mark server-info plugin as deprecated [#12244](https://github.com/apache/apisix/pull/12244)
+- feat(consumer): consumer username allows - in it [#12296](https://github.com/apache/apisix/pull/12296)
+- refactor: change log level to debug to avoid unnecessary logs [#12361](https://github.com/apache/apisix/pull/12361)
+- chore: change log level from warn to info for stale batch processor removal [#12297](https://github.com/apache/apisix/pull/12297)
+- feat(standalone): allow more characters in credential_id for API-driven mode [#12295](https://github.com/apache/apisix/pull/12295)
```

Comment:
API-driven mode is also new in this release and changes in this PR will not be breaking changes for users.